Assets/Editor/CreatureHydrator.cs

```diff
@@ -218,25 +218,28 @@ private static string GenerateScriptContent(string unitName, string generaName,
         return $@"
             using UnityEngine;
             using Creatures.Api;
-
-            public class {unitName}Script : Creature, {generaName}
+            
+            namespace Creatures.impl
             {{
-                public string GeneraType => ""{generaName}"";
-                
-                void Start()
+                public class {unitName}Script : Creature, {generaName}
                 {{
-                    Speed = {speed}f;
-                    HealthScript = new HealthScript({health});
+                    public string GeneraType => ""{generaName}"";
                     
-                    // Add custom initialization logic here
-                }}
-
-                // Optional: Override the Update method if custom logic is needed
-                protected override void Update()
-                {{
-                    base.Update();
-
-                    // Add custom update logic here, if necessary
+                    void Start()
+                    {{
+                        Speed = {speed}f;
```

Comment:
I wonder how simple we can make this script; i.e. if we can make it read from the YAML files directly and inject things, so this basically is just a list of some file paths to load?

Reply:
Oh yeah, I know my template here is going to need to change ... I think I would prefer if there was some template format